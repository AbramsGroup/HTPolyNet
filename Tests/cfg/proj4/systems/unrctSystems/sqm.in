Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   8     O       -0.1470        1.3670       -0.2320 
   6     C       -0.9000        1.0270        0.7890 
   6    C3       -0.1260        0.4150        1.7530 
   6    C4        1.0910        1.0020        0.0150 
   6    C5        1.1580        0.4000        1.2590 
   6    C6        2.2230        1.2420       -0.9450 
   7     N        3.5020        0.6910       -0.4220 
   1     H        2.2590        2.3440       -1.0860 
   1    H1        1.9750        0.8210       -1.9440 
   6    C7        4.5900        1.6930       -0.5680 
   6    C8        3.8610       -0.6060       -1.0640 
   6    C9        2.9150       -1.7740       -0.7000 
   1    H2        4.8570       -0.9270       -0.7060 
   1    H3        3.9310       -0.4910       -2.1700 
   6    C1        3.4590       -3.0940       -1.2540 
   1    H4        3.5280       -3.0510       -2.3620 
   1    H5        2.7840       -3.9290       -0.9700 
   1    H6        4.4670       -3.3040       -0.8370 
   8    O1        1.6330       -1.5620       -1.2310 
   1    H7        2.8730       -1.8470        0.4110 
   6   C10        5.8390        1.3870        0.2950 
   1    H8        4.2260        2.6860       -0.2170 
   1    H9        4.8800        1.8000       -1.6380 
   8    O2        6.7970        2.4030        0.1310 
   6    C2        5.4880        1.2210        1.7790 
   1   H10        6.3210        0.4490       -0.0420 
   1   H11        5.0220        2.1470        2.1790 
   1   H12        6.4100        1.0040        2.3610 
   1   H13        4.7820        0.3750        1.9190 
   1   H14        6.4130        3.2470        0.4890 
   1   H15        0.9990       -2.0890       -0.6780 
   1   H16       -1.9650        1.2070        0.8510 
   1   H17       -0.4610        0.0270        2.7050 
   1   H18        2.0320       -0.0020        1.7530 

