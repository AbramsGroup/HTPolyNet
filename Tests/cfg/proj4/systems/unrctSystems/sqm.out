            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 43

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       79
| QMMM:    diag iterations used for timing =       15
| QMMM:
| QMMM:              Internal diag routine = 0.015469 seconds
| QMMM:                 Dspev diag routine = 0.017667 seconds
| QMMM:                Dspevd diag routine = 0.011941 seconds
| QMMM:                Dspevx diag routine = 0.061601 seconds
| QMMM:                 Dsyev diag routine = 0.017802 seconds
| QMMM:                Dsyevd diag routine = 0.017541 seconds
| QMMM:                Dsyevr diag routine = 0.018084 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.005666 seconds
| QMMM:
| QMMM: Using dspevd routine (diag_routine=3).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      O       -0.1470    1.3670   -0.2320
  QMMM:     2        2      C       -0.9000    1.0270    0.7890
  QMMM:     3        3      C       -0.1260    0.4150    1.7530
  QMMM:     4        4      C        1.0910    1.0020    0.0150
  QMMM:     5        5      C        1.1580    0.4000    1.2590
  QMMM:     6        6      C        2.2230    1.2420   -0.9450
  QMMM:     7        7      N        3.5020    0.6910   -0.4220
  QMMM:     8        8      H        2.2590    2.3440   -1.0860
  QMMM:     9        9      H        1.9750    0.8210   -1.9440
  QMMM:    10       10      C        4.5900    1.6930   -0.5680
  QMMM:    11       11      C        3.8610   -0.6060   -1.0640
  QMMM:    12       12      C        2.9150   -1.7740   -0.7000
  QMMM:    13       13      H        4.8570   -0.9270   -0.7060
  QMMM:    14       14      H        3.9310   -0.4910   -2.1700
  QMMM:    15       15      C        3.4590   -3.0940   -1.2540
  QMMM:    16       16      H        3.5280   -3.0510   -2.3620
  QMMM:    17       17      H        2.7840   -3.9290   -0.9700
  QMMM:    18       18      H        4.4670   -3.3040   -0.8370
  QMMM:    19       19      O        1.6330   -1.5620   -1.2310
  QMMM:    20       20      H        2.8730   -1.8470    0.4110
  QMMM:    21       21      C        5.8390    1.3870    0.2950
  QMMM:    22       22      H        4.2260    2.6860   -0.2170
  QMMM:    23       23      H        4.8800    1.8000   -1.6380
  QMMM:    24       24      O        6.7970    2.4030    0.1310
  QMMM:    25       25      C        5.4880    1.2210    1.7790
  QMMM:    26       26      H        6.3210    0.4490   -0.0420
  QMMM:    27       27      H        5.0220    2.1470    2.1790
  QMMM:    28       28      H        6.4100    1.0040    2.3610
  QMMM:    29       29      H        4.7820    0.3750    1.9190
  QMMM:    30       30      H        6.4130    3.2470    0.4890
  QMMM:    31       31      H        0.9990   -2.0890   -0.6780
  QMMM:    32       32      H       -1.9650    1.2070    0.8510
  QMMM:    33       33      H       -0.4610    0.0270    2.7050
  QMMM:    34       34      H        2.0320   -0.0020    1.7530

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -101.9723 kcal/mol        0.6115 kcal/(mol*A)
